package.json

```diff
@@ -14,13 +14,13 @@
     "@sap/xb-msg-amqp-v100": "^0.9.53",
     "@sap/xssec": "^4",
     "express": "^4",
-    "passport": "^0.4.1"
+    "passport": "^0.4.1",
+    "@cap-js/sqlite":"^1.0.1"
```

Comment:
for the main application, "@cap-js/sqlite" to be part of devDependencies as we use HANA for deployment.